### PR TITLE
share meta tags.

### PR DESCRIPTION
**Pull request by Tulun**
Metatags for Across.

Signed-off-by: Tulun <jaykiraly@gmail.com>

### CODE REVIEW

**Comment by Tulun** [comment]
Screen Shot 2021-11-09 at 4 00 33 PM
@daywiss  ;)

**Review by daywiss — APPROVED** [review]

File: public/index.html [hunk 1]
```diff
@@ -9,13 +9,29 @@
...
     <meta
       name="google-site-verification"
       content="xs1HS4-nzd-a2ROd3-pQsn35pCc1WwMnlE4uK032IJ0"
     />
+    <meta name="robots" content="index, follow" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo-small.png" />
-    <title>Across</title>
+    <title>Across - Bridge L1 and L2 Ethereum assets</title>
```
Comment (Tommy1231232) [inline public/index.html:34]: Update to "Bridge Layer 1 and Layer 2 assets"

File: public/index.html [hunk 2]
```diff
@@ -9,13 +9,29 @@
...
+    <meta property="og:title" content="Across" />
+    <meta
+      property="og:description"
+      content="Across is the fastest, cheapest and most secure crosschain bridge."
+    />
+    <meta property="og:image" content="%PUBLIC_URL%/logo-small.png" />
+    <meta name="twitter:site" content="Across" />
+    <meta
+      name="twitter:card"
+      content="Across is the fastest, cheapest and most secure crosschain bridge."
```
Comment (Tommy1231232) [inline public/index.html:21]: Update "crosschain" to "cross-chain"

File: public/index.html [hunk 3]
```diff
@@ -9,13 +9,29 @@
       name="description"
       content="Across is the fastest, cheapest and most secure crosschain bridge."
     />
+    <meta property="og:title" content="Across" />
+    <meta
+      property="og:description"
+      content="Across is the fastest, cheapest and most secure crosschain bridge."
```
Comment (Tommy1231232) [inline public/index.html:15]: update crosschain to be cross-chain

**Review by Tommy1231232 — CHANGES_REQUESTED** [review]

**Pull request merged**